### PR TITLE
Settings

### PR DESCRIPTION
**Pull request by oneilsw**
-Created The Settings button on the Home Screen.
-Created SettingsScreen.js, PasswordChangeScreen.js, AudioScreen.js, and EnviornmentChangeScreem.js
-Created 3 buttons on the Settings Screen routed to Password Change Screen, Audio Screen, and Environment Change Screen

#23 

Once this has been approved and merged into master, we can create seperate issues for Password change, audio, and environment change

### CODE REVIEW

**Comment by ali1195shah** [comment]
I'll get on it tonight 👍

File: Big2/src/screens/AudioScreen.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+
+const AudioScreen = () => {
+  return (
+    <View style={styles.center}>
+      <Text>Audio Screen</Text>
+    </View>
+  )
+}
+
+export default AudioScreen;
```
Comment (pavel-ilin) [inline Big2/src/screens/AudioScreen.js:13]: Do we need Audio screen? I don't say we don't, but I remember we had a discussion and @bcsaxon gave as good argument about how usually players change volume in the games (by pressing physical sound button on the phone). Let's make a decision.

All the rest, I think LGTM.

Comment (oneilsw) [reply Big2/src/screens/AudioScreen.js:13]: The audio screen is to change the different background music and sound effects. not necessarily to change the volume.

Comment (pavel-ilin) [reply Big2/src/screens/AudioScreen.js:13]: Cool! Then LGTM!

**Pull request merged**